README.md

```diff
@@ -49,7 +52,10 @@ add her to the list of guests in the castle.
 
 Write your code here:
 ```ruby
-# code here
+town[:residents][1]
+town[:residents].delete("Belle")
+town[:castle][:guests].push("Belle")
+town[:castle][:guests]
 ```
```

Comment:
:+1: Lines 55 & 58 aren't necessary, but it's great to see you checking your work! Programmers can get carried away with the, "Move fast and break stuff" ethos, but the world needs "Measure twice, cut once" developers, too!